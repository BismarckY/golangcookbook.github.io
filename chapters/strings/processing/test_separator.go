package main

import (
	"fmt"
	"strings"
)

func main() {
	s := "root:*:0:0:System Administrator:/root:/bin/sh"
	words := strings.Split(s, ":")
	for i, word := range words {
		fmt.Println(i, " => ", word)
	}
}
