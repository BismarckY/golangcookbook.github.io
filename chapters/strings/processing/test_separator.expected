0  =>  root
1  =>  *
2  =>  0
3  =>  0
4  =>  System Administrator
5  =>  /root
6  =>  /bin/sh
